<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Greedy_Triangulation_Dialog</class>
 <widget class="QWidget" name="Greedy_Triangulation_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Greedy Triangulation Setup</string>
  </property>
  <widget class="QSpinBox" name="spinBox_KSearch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_KSearch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>KSearch</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SearchRadius">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SearchRadius</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_SearchRadius">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Mu</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Mu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_MaximumNearestNeighbors">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MaximumNearestNeighbors</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_MaximumNearestNeighbors">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_MaximumSurfaceAngle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MaximumSurfaceAngle</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_MaximumSurfaceAngle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_MinimumAngle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MinimumAngle</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_MinimumAngle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_MinimumAngle_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MaximumAngle</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_MaximumAngle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_NormalConsistency">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>161</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>NormalConsistency</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
